Source/WTF/wtf/MemoryPressureHandler.cpp

```diff
@@ -40,7 +40,7 @@ namespace WTF {
 
 WTF_EXPORT_PRIVATE bool MemoryPressureHandler::ReliefLogger::s_loggingEnabled = false;
 
-#if PLATFORM(IOS_FAMILY)
+#if PLATFORM(IOS_FAMILY) || PLATFORM(WPE)
```

Comment:
We cannot change the default values of the fractions, cause this affects all of the devices. As I told you in previous PRs, there's API to change these values for your platform only, please use it.